en/04_Changelogs/5.2.0.md

```diff
@@ -172,6 +173,33 @@ The files section of the CMS now has buttons to select and deselect all files an
 
 ![select all functionality in the CMS](../_images/asset-admin-select-all.png)
 
+### Support for validation using `symfony/validator`{#symfony-validator}
+
+A new [`ConstraintValidator`](api:SilverStripe\Core\Validation\ConstraintValidator) class has been added which provides an abstraction around [`symfony/validator`](https://symfony.com/doc/current/components/validator.html), so you can easily validate values against symfony's validation constraints and get a [`ValidationResult`](api:SilverStripe\ORM\ValidationResult) object as a result.
+
+```php
+use SilverStripe\Core\Validation\ConstraintValidator;
+
+/**
+ * @var \Symfony\Component\Validator\Constraint $constraint
+ * @var \SilverStripe\ORM\ValidationResult $result
```

Comment:
```suggestion
 * @var Symfony\Component\Validator\Constraint $constraint
 * @var SilverStripe\ORM\ValidationResult $result
```

Reply:
Needs the backslash because otherwise it'll append to the end of any namespace the developer's using.

We probably have existing docs that need a leading backslash, but I'll open a separate card for that.